lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -728,6 +730,7 @@ def get_daterange_results(
             ("drb", {"owner": "drb"}, ["drb", "fio_1"]),
             ("drb", {"mine": "true"}, ["drb", "fio_1"]),
             ("drb", {"access": "public"}, ["fio_1", "fio_2"]),
+            ("drb", {"name": "noname"}, []),
```

Comment:
Do we also have a test case where the result is empty due to the choice of dates?  (If not...should we?)